src/main/java/com/remondis/cdc/consumer/pactbuilder/ConsumerBuilderImpl.java

```diff
@@ -20,6 +20,8 @@
 import au.com.dius.pact.consumer.dsl.DslPart;
 import au.com.dius.pact.consumer.dsl.PactDslJsonBody;
 import au.com.dius.pact.consumer.dsl.PactDslJsonRootValue;
+import com.fasterxml.jackson.databind.introspect.AnnotatedMethod;
+import com.fasterxml.jackson.databind.util.BeanUtil;
 
```

Comment:
Use `nonNull()` instead of `!isNull()`
```suggestion
import static java.util.Objects.nonNull;
```